www/themes/borg/css/page-project-search.css

```diff
@@ -229,7 +229,7 @@ form.views-exposed-form,
   padding: 0;
 }
 
-.result__info li {
+.result__info > li {
```

Comment:
These CSS changes will need be applied to the borg theme.  They can either 1) be merged, or 2) a patch from the PR can be applied here, and documented in `PATCHS.txt` before this PR can be merged.

Or, if this CSS is specific to only the b.org sites, it can be moved into the subtheme.  :)